### PR TITLE
Document the new storage directory structure

### PR DESCRIPTION
**Pull request by MattF-NSIDC**
## Description

Document the new storage directory structure

## Checklist

If an item on this list is done _or not needed_, simply check it with `[x]`.

- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build`)
- [x] CHANGELOG.md updated
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

File: doc/CONTRIBUTING.md [hunk 1]
```diff
@@ -9,6 +9,71 @@ release except releases labeled as "stable". Stable releases can be found at
...
+read-write.
+
+
+#### Dataset fetch storage
+
+```
+data/working-storage/fetch-datasets
+```
+
+Datasets are fetched for their original locations and written to this
```
Comment (trey-stafford) [inline doc/CONTRIBUTING.md:38]: `fetched for` -> `fetched from`

File: doc/CONTRIBUTING.md [hunk 2]
```diff
@@ -9,6 +9,71 @@ release except releases labeled as "stable". Stable releases can be found at
...
+
+
+### "Private archive" storage
+
+```
+data/private-archive
+```
+
+While we prefer to only include publicly-archived data, we do have some
+privately archived data that we've sourced by e-mailing scientists or manually
```
Comment (trey-stafford) [inline doc/CONTRIBUTING.md:69]: `we've` -> `we have`. Or maybe `data that **have been** sourced`.

File: doc/CONTRIBUTING.md [hunk 3]
```diff
@@ -9,6 +9,71 @@ release except releases labeled as "stable". Stable releases can be found at
...
+Finalized layers and projects are stored in these directories.
+
+
+### "Private archive" storage
+
+```
+data/private-archive
+```
+
+While we prefer to only include publicly-archived data, we do have some
```
Comment (trey-stafford) [inline doc/CONTRIBUTING.md:68]: `While we` -> `While this project prefers`?

File: doc/CONTRIBUTING.md [hunk 4]
```diff
@@ -9,6 +9,71 @@ release except releases labeled as "stable". Stable releases can be found at
...
+privately archived data that we've sourced by e-mailing scientists or manually
+interacting with machine-unfriendly systems. These datasets have an
+`access_instructions` attribute in configuration that describes how the data
+was acquired.
```
Comment (trey-stafford) [inline doc/CONTRIBUTING.md:72]: Maybe worth noting the flag that excludes private archive data? Point to that part of the docs?

**Review by trey-stafford — CHANGES_REQUESTED** [review]
Some suggestions but otherwise looks good to me!

File: doc/CONTRIBUTING.md (hunk 1 above)
Comment (MattF-NSIDC) [reply doc/CONTRIBUTING.md]: 03b3ecc

File: doc/CONTRIBUTING.md (hunk 3 above)
Comment (MattF-NSIDC) [reply doc/CONTRIBUTING.md]: 1b8e9b1

File: doc/CONTRIBUTING.md (hunk 4 above)
Comment (MattF-NSIDC) [reply doc/CONTRIBUTING.md]: 1218994

**Review by trey-stafford — APPROVED** [review]

**Pull request merged**